doc/releases/release-notes-3.6.rst

```diff
@@ -456,6 +456,13 @@ Drivers and Sensors
   * Added support for optical encoders and power management to :dtcompatible:`gpio-qdec`.
   * New drivers :dtcompatible:`espressif,esp32-touch`, :dtcompatible:`analog-axis`.
 
+* MFD
+
+  * Added support for :dtcompatible:`maxim,max20335`
+  * Added support for :dtcompatible:`adi,ad5592`
+  * Added separate initialisation priorities for :dtcompatible:`nordic,npm1300` and
```

Comment:
nit (i need to look at how other projects do US vs UK English at some point. Really not that important, but somehow it feels like it's better to keep things consistent and it certainly looks like Zephyr code base is leaning towards US English)

```suggestion
  * Added separate initialization priorities for :dtcompatible:`nordic,npm1300` and
```